tekton/src/tasks/gitops/gitops-provision-mas-fvt-core.yml.j2

```diff
@@ -0,0 +1,120 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gitops-provision-mas-fvt-core
+spec:
+  params:
+    - name: cluster_name
+      type: string
+    - name: mas_instance_id
+      type: string
+    - name: pipeline_storage_class
+      type: string
+    - name: aws_region
+      type: string
+    - name: aws_secret_access_key
+      type: string
+    - name: aws_access_key_id
+      type: string
+    - name: fvt_pipeline_name
+      type: string
+    - name: cli_version
+      type: string
+    - name: fvt_artifactory_username
+      type: string
+    - name: fvt_artifactory_token
+      type: string
+    - name: fvt_config
+      type: string
+    - name: fvt_image_registry
+      type: string
+    - name: fvt_version_core
+      type: string
+    - name: ivt_version_core
+      type: string
+
+  stepTemplate:
+    name: gitops-provision-mas-fvt-core
+    env:
+      - name: CLUSTER_NAME
+        value: $(params.cluster_name)
+      - name: MAS_INSTANCE_ID
+        value: $(params.mas_instance_id)
+      - name: PIPELINE_STORAGE_CLASS
+        value: $(params.pipeline_storage_class)
+      - name: AWS_REGION
+        value: $(params.aws_region)
+      - name: AWS_SECRET_ACCESS_KEY
+        value: $(params.aws_secret_access_key)
+      - name: AWS_ACCESS_KEY_ID
+        value: $(params.aws_access_key_id)
+      - name: FVT_PIPELINE_NAME
+        value: $(params.fvt_pipeline_name)
+      - name: CLI_VERSION
+        value: $(params.cli_version)
+      - name: FVT_ARTIFACTORY_USERNAME
+        value: $(params.fvt_artifactory_username)
+      - name: FVT_ARTIFACTORY_TOKEN
+        value: $(params.fvt_artifactory_token)
+      - name: FVT_CONFIG
+        value: $(params.fvt_config)
+      - name: FVT_IMAGE_REGISTRY
+        value: $(params.fvt_image_registry)
+      - name: FVT_VERSION_CORE
+        value: $(params.fvt_version_core)
+      - name: IVT_VERSION_CORE
+        value: $(params.ivt_version_core)
+  steps:
+    - args:
+      - |-
+
+        export ROSA_CONFIG=/workspace/configs/tmp-rosa-init/rosa-$(params.cluster_name)-details.yaml
+        export OCP_SERVER=$(cat $ROSA_CONFIG | yq -r '.data.api_url')
+        export OCP_USERNAME=$(cat $ROSA_CONFIG | yq -r '.data.username')
+        export ROSA_CLUSTER_ADMIN_PASSWORD=$(cat $ROSA_CONFIG | yq -r '.data.admin_password')
+        oc login -u $OCP_USERNAME -p $ROSA_CLUSTER_ADMIN_PASSWORD $OCP_SERVER --insecure-skip-tls-verify
+        OC_RC=$?
+        if [ $OC_RC -ne 0 ]; then
+          echo "oc login failed, exiting with error.."
+          exit 1
+        fi
+
+        echo "Set k8s context"
+        export K8S_AUTH_CONTEXT=$(oc whoami -c)
+
+        export MAS_CONFIG_DIR=/tmp/additional-configs
+        mkdir -p $MAS_CONFIG_DIR
+
+        if [[ -e /workspace/shared-entitlement/entitlement.lic ]]
+          cat /workspace/shared-entitlement/entitlement.lic > /tmp/additional-configs/entitlement.lic
```

Comment:
Can we just cp all files under /workspace/shared-entitlement and /workspace/shared-additional-configs to the working dir /tmp/additional-configs ? this would then allow for new configs to be added and not have to worry about changing this.